### PR TITLE
Update local settings template with default obs id

### PR DESCRIPTION
**Pull request by CFarcy**

### CODE REVIEW

File: src/project/settings/local.py.dist [hunk 1]
```diff
@@ -10,5 +10,6 @@ SECRET_KEY = 'super-hot-secret-123456789'
 
 # Create the obsevatory layer, update next next line and uncomment it
```
Comment (jrmi) [inline src/project/settings/local.py.dist:11]:
```suggestion
# Create the obsevatory layer, update the two next lines and uncomment them
```

File: src/project/settings/local.py.dist [hunk 2]
```diff
@@ -8,7 +8,8 @@
 
 SECRET_KEY = 'super-hot-secret-123456789'
 
-# Create the obsevatory layer, update next next line and uncomment it
+# Create the obsevatory layer, update the two next lines and uncomment them
 # TROPP_OBSERVATORY_LAYER_PK = 1
+# TROPP_OBSERVATORY_ID = 20
```
Comment (jrmi) [inline src/project/settings/local.py.dist:13]: Add default value or add waring at startup like for obs id

Comment (CFarcy) [reply src/project/settings/local.py.dist:13]: Checks and warnings have been added to `terra-opp`. Is it enough?

Comment (jrmi) [reply src/project/settings/local.py.dist:13]: Yes!

**Review by jrmi — APPROVED** [review]

**Pull request merged**